### PR TITLE
maze_runner benchmark: fix appearance of table

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
Restore the maze drawing on the table top.

maze_runner_2

### CODE REVIEW

File: projects/samples/robotbenchmark/maze_runner/worlds/maze_runner.wbt [hunk 1]
```diff
@@ -51,6 +51,21 @@ Table {
...
+  trayAppearance PBRAppearance {
+    baseColorMap ImageTexture {
+      url [
+        "textures/table_top.jpg"
+      ]
+    }
+    roughness 1
+    metalness 0
+    textureTransform TextureTransform {
+      scale 1 1.7999999999999998
```
Comment (omichel) [inline projects/samples/robotbenchmark/maze_runner/worlds/maze_runner.wbt]:
```suggestion
      scale 1 1.8
```

Comment (stefaniapedrazzi) [reply projects/samples/robotbenchmark/maze_runner/worlds/maze_runner.wbt]: Thank you.
I set it to 1.8 from the GUI, but this has been automatically changed to 1.7999999999999998 when saving the file.

Comment (omichel) [reply projects/samples/robotbenchmark/maze_runner/worlds/maze_runner.wbt]: Strange, I thought we fixed that problem some time ago...

Comment (omichel) [reply projects/samples/robotbenchmark/maze_runner/worlds/maze_runner.wbt]: I would be nice to report it in a new (or old) issue.

Comment (stefaniapedrazzi) [reply projects/samples/robotbenchmark/maze_runner/worlds/maze_runner.wbt]: I will try to reproduce it and report it

**Comment by stefaniapedrazzi** [comment]
:+1:

**Pull request merged**